pose-detection/src/blazepose_tfjs/detector.ts

```diff
@@ -326,14 +328,42 @@ export class BlazePoseTfjsDetector extends BasePoseDetector {
     // score.
     // Output [2]: This tensor (shape: [1, 64, 64, 39]) represents heatmap for
     // the 39 landmarks.
+    // Lite model:
+    // Output[1]: This tensor (shape: [1, 195]) represents 39 5-d keypoints.
+    // Output[2]: This tensor (shape: [1, 1]) represents the confidence score.
+    // Output[4]: This tensor (shape: [1, 64, 64, 39]) represents heatmap for
+    // the 39 landmarks.
+    // Heavy model:
+    // Output[3]: This tensor (shape: [1, 195]) represents 39 5-d keypoints.
+    // Output[2]: This tensor (shape: [1, 1]) represents the confidence score.
+    // Output[4]: This tensor (shape: [1, 64, 64, 39]) represents heatmap for
+    // the 39 landmarks.
     const landmarkResult =
         this.landmarkModel.predict(imageValueShifted) as tf.Tensor[];
 
     let landmarkTensor, poseFlagTensor, heatmapTensor;
 
-    landmarkTensor = landmarkResult[3] as tf.Tensor2D;
-    poseFlagTensor = landmarkResult[0] as tf.Tensor2D;
-    heatmapTensor = landmarkResult[2] as tf.Tensor4D;
+    switch (this.modelType) {
+      case 'lite':
+        landmarkTensor = landmarkResult[1] as tf.Tensor2D;
```

Comment:
In general, it is better to have unified order of output tensors for models, if converter supports this.